inst/include/plant/environment.h

```diff
@@ -42,20 +42,20 @@ class Environment {
 
   void clear_environment() {}
 
-  double seed_rain_dt() const {
-    if (seed_rain.empty()) {
-      Rcpp::stop("Cannot get seed rain for empty environment");
+  double offspring_arriving_dt() const {
```

Comment:
Why are we using  `offspring_arriving_dt` here instead of `birth_rate_dt`? The new term may be confusing. 

Also, are we moving the broth_rate stuff out of environment? Or is that in next step, after successful renaming